packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -548,4 +443,169 @@ void main() {
     );
     await gesture.up();
   });
+
+  testWidgets('The slider track height can be overridden', (WidgetTester tester) async {
+    final SliderThemeData sliderTheme = ThemeData().sliderTheme.copyWith(trackHeight: 16);
+
+    await tester.pumpWidget(_buildApp(sliderTheme, value: 0.25));
+
+    final RenderBox sliderBox = tester.firstRenderObject<RenderBox>(find.byType(Slider));
+
+    // Top and bottom are centerY (300) + and - trackRadius (8).
+    expect(
+      sliderBox,
+      paints
+        ..rect(rect: Rect.fromLTRB(16.0, 292.0, 208.0, 308.0), color: sliderTheme.activeTrackColor)
+        ..rect(rect: Rect.fromLTRB(208.0, 292.0, 784.0, 308.0), color: sliderTheme.inactiveTrackColor)
+    );
+
+    await tester.pumpWidget(_buildApp(sliderTheme, value: 0.25, enabled: false));
+    await tester.pumpAndSettle(); // wait for disable animation
+
+    // The disabled thumb is smaller so the active track has to paint longer to
```

Comment:
I think you mean that the track segments before and after the thumb are longer?

Reply:
They are shorter*, this was worded wrong originally. The point is that the segments are actually shorter because of the gap, where as in the enabled state it paints the entire track to the center of the thumb (underneath it). 

Please see the new comment above the enabled state expect() and the rephrased paragraph above the disabled state expect()